content/en/docs/concepts/security/mutating-webhooks-good-practices.md

```diff
@@ -0,0 +1,585 @@
+---
+title: Mutating Webhook Good Practices
```

Comment:
Here's the dream:
```suggestion
title: Admission Webhook Good Practices
```

What's the smallest piece of work we can do to make this applicable to both validating and mutating admission webhooks?

Reply:
I was really leaning towards this as I wrote it. A lot of this is applicable to both, and we can still explicitly call out when advice applies to just mutating. I'll rework wording on the page soon to try and make this agnostic